rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -315,7 +563,11 @@ void memory_viewer_panel::scroll(s32 steps)
 	m_addr -= m_addr % (m_colcount * 4); // Align by amount of bytes in a row
 
 	m_addr_line->setText(qstr(fmt::format("%08x", m_addr)));
-	ShowMemory();
+
+	QTimer::singleShot(0, [this]()
```

Comment:
why

Reply:
Removed here, in resize it vis for safety to ensure ther won't be recusrion.